website/server/controllers/api-v3/tasks.js

```diff
@@ -920,10 +1144,13 @@ api.clearCompletedTodos = {
  *
  * @apiParam {String} taskId The task _id or alias
  *
+ * @apiExample {json} Example call:
+ * curl -X "DELETE" https://habitica.com/api/v3/tasks/3d5d324d-a042-4d5f-872e-0553e228553e
+ *
  * @apiSuccess {Object} data An empty object
  *
  * @apiUse TaskNotFound
- * @apiUse ChallengeNotFound
+ * @apiError (400) {NotAuthorized} Challenge A task belonging to a challenge can't be deleted.
```

Comment:
added errors for unauthorized group, unauthorized challenge and can't delete group task